### ISSUE TITLE
Добавить в интерфейс раннера кнопку смены вида [V]

### ISSUE THREAD

**Issue by oPOCCOMAXAo** [MailRuChamps/raic-2020#94]
Для мобильных устройств это ускорит воспроизведение. На них нажать клавишу клавиатуры нельзя, так что кнопка в интерфейсе послужит заменой

**Comment by tyamgin** [comment]
Да, бесит. Оригинальный вид вообще ужасный и неюзабельный. Синее на синем.

**Comment by oPOCCOMAXAo** [comment]
For data deletion
